pennylane/operation.py

```diff
@@ -423,6 +433,8 @@ class Operation(Operator):
     """
     # pylint: disable=abstract-method
     _grad_recipe = None
+    string_for_inverse = ".inv"
+    _inverse = False
```

Comment:
Where is this property used?

Reply:
Great one! As you hinted on in other comments, I have modified the functionality of the ``inv()`` method.